internal/utils/strings.go

```diff
@@ -30,6 +30,36 @@ func SliceString(s string, d int) (array []string) {
 	return
 }
 
+// IsStringSlicesDifferent checks two slices of strings and on the first occurrence of a difference returns true.
```

Comment:
You should document whether sorting is important. It would avoid reading the code to get it.